cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,89 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
+const faker = require("faker");
 
-  });
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import homePageObject from '../support/pages/home.pageObject';
+import settingsPageObject from '../support/pages/settings.pageObject.js'
+import ProfilePageObject from '../support/pages/profile.pageObject.js'
 
-  beforeEach(() => {
+const signInPage = new SignInPageObject();
+const homePage = new homePageObject();
+const settingsPage = new settingsPageObject();
+const profilePage = new ProfilePageObject();
 
+describe('Settings page', () => {
+  let user;
+ 
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
   });
 
   it('should provide an ability to update username', () => {
-
+    const newUsername = faker.name.firstName();
+    
+    cy.login(user.email, user.username, user.password);
+    settingsPage.visit();
+    settingsPage.fillUsernameField(newUsername.toLowerCase());
+    settingsPage.clickUpdateBtn();
+    homePage.assertHeaderContainUsername(newUsername.toLowerCase());
   });
 
   it('should provide an ability to update bio', () => {
-
+    const newBio = faker.random.word();
```

Comment:
I would advise you to use words, because the bio usually consists of sentences, not one word

Reply:
agree